dolphinscheduler-ui/src/views/projects/task/components/node/format-data.ts

```diff
@@ -658,5 +664,10 @@ export function formatModel(data: ITaskData) {
   if (data.taskParams?.jobType) {
     params.isCustomTask = data.taskParams.jobType === 'CUSTOM'
   }
+
+  if (data.taskParams?.type) {
+    params.type = data.taskParams.type
```

Comment:
It seems that the type doesn't need to be reassigned here.

Reply:
Sure I will remove it.